src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -111,6 +111,11 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         /// has no concept of unmanaged types.
         /// </summary>
         bool IsUnmanagedType { get; }
+
+        /// <summary>
+        /// True if the type is a readonly struct.
```

Comment:
The assumption being that if we ever had readonly classes we'd just change the comment?

Reply:
That's what I was going for.

Reply:
Do we envision a future scenario where a non-NamedTypeSymbol can be readonly? If not, should this property be exposed on `INamedTypeSymbol`?

Reply:
I believe so, you could have a readonly array type.